yaptide/routes/common_sim_routes.py

```diff
@@ -139,6 +139,28 @@ def get_all_estimators(sim_id: int):
                             content={"estimators": result_estimators})
 
 
+def create_or_update_estimator(sim_id: int, name: str, estimator_dict: dict):
+    estimator = fetch_estimator_by_sim_id_and_file_name(sim_id=sim_id, file_name=estimator_dict["name"])
+    if not estimator:
+        estimator = EstimatorModel(name=name, file_name=estimator_dict["name"], simulation_id=sim_id)
+        estimator.data = estimator_dict["metadata"]
+        add_object_to_db(estimator)
+
+    for page_dict in estimator_dict["pages"]:
+        page = fetch_page_by_est_id_and_page_number(est_id=estimator.id,
+                                                    page_number=int(page_dict["metadata"]["page_number"]))
+
+        page_existed = bool(page)
+        if not page_existed:
+            # create new page
+            page = PageModel(page_number=int(page_dict["metadata"]["page_number"]), estimator_id=estimator.id)
+        # we always update the data
+        page.data = page_dict
+        if not page_existed:
+            # if page was created, we add it to the session
+            add_object_to_db(page, False)
```

Comment:
```suggestion
            add_object_to_db(page, make_commit=False)
```

its hard to guess what False mean if you don't provide argument name